homeassistant/components/homekit/type_media_players.py

```diff
@@ -142,3 +170,211 @@ def update_state(self, new_state):
                               self.entity_id, current_state)
                 self.chars[FEATURE_TOGGLE_MUTE].set_value(current_state)
             self._flag[FEATURE_TOGGLE_MUTE] = False
+
+
+@TYPES.register('TelevisionMediaPlayer')
+class TelevisionMediaPlayer(HomeAccessory):
+    """Generate a Television Media Player accessory."""
+
+    def __init__(self, *args):
+        """Initialize a Switch accessory object."""
+        super().__init__(*args, category=CATEGORY_SWITCH)
+        self._flag = {FEATURE_ON_OFF: False, FEATURE_PLAY_PAUSE: False,
+                      FEATURE_TOGGLE_MUTE: False, FEATURE_SELECT_SOURCE: False}
+        self.chars = {FEATURE_ON_OFF: None, FEATURE_PLAY_PAUSE: None,
+                      FEATURE_TOGGLE_MUTE: None, FEATURE_SELECT_SOURCE: None,
+                      FEATURE_VOLUME_STEP: None}
+        feature_list = self.config[CONF_FEATURE_LIST]
+
+        self.sources = []
+
+        self.category = CATEGORY_TELEVISION
+        television = self.add_preload_service(SERV_TELEVISION,
+                                              [
+                                                  CHAR_CONFIGURED_NAME,
+                                                  CHAR_REMOTE_KEY
+                                              ])
+        television.configure_char(CHAR_CONFIGURED_NAME,
+                                  value=self.display_name)
+        television.configure_char(CHAR_SLEEP_DISCOVER_MODE, value=True)
+        self.chars[FEATURE_ON_OFF] = television.configure_char(
+            CHAR_ACTIVE, setter_callback=self.set_on_off)
+
+        if FEATURE_PLAY_PAUSE in feature_list:
+            self.chars[FEATURE_PLAY_PAUSE] \
+                    = television.configure_char(
+                        CHAR_REMOTE_KEY,
+                        setter_callback=self.set_remote_key)
+
+        if FEATURE_TOGGLE_MUTE in feature_list:
+
+            television_speaker = self.add_preload_service(
+                SERV_TELEVISION_SPEAKER, [
+                    CHAR_NAME, CHAR_ACTIVE,
+                    CHAR_VOLUME_CONTROL_TYPE,
+                    CHAR_VOLUME_SELECTOR,
+                    CHAR_VOLUME
+                ])
+            television.add_linked_service(television_speaker)
+
+            name = '{} {}'.format(self.display_name, 'Volume')
+            television_speaker.configure_char(CHAR_NAME, value=name)
```

Comment:
`name` is only used here, isn't it? Just insert it directly. No need to create a new variable.